### ISSUE TITLE
[Bug]: Docker suddenly hangs up by itself

### ISSUE THREAD

**Issue by rolanzocnyang** [milvus-io/milvus#23275]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.2.3
- Deployment mode(standalone or cluster): standalone 
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus 2.2.3
- OS(Ubuntu or CentOS): Ubuntu 
- CPU/Memory: 32GB
- GPU: NO
- Others:
```

### Current Behavior

After several days of MILVUS running, Docker suddenly hangs up by itself. I think the time of occurrence is 2023/04/02 01:03:10.560.

### Milvus Log

[milvus-etcd_log_0402.txt](https://github.com/milvus-io/milvus/files/11175906/milvus-etcd_log_0402.txt)
[milvus-standalone_log_0402.txt](https://github.com/milvus-io/milvus/files/11175928/milvus-standalone_log_0402.txt)

**Comment by yanliang567** [comment]
I doubt that the etcd service was down at that time. could you please double check that?
`[2023/04/02 00:57:21.787 +00:00] [ERROR] [config/etcd_source.go:151] ["can not pull configs"] [error="context deadline exceeded"] [stack="github.com/milvus-io/milvus/internal/config.(*EtcdSource).refreshConfigurationsPeriodically\n\t/go/src/github.com/milvus-io/milvus/internal/config/etcd_source.go:151"]
`

/assign @rolanzocnyang 
/unassign

**Comment by rolanzocnyang** [comment]
Thanks for your reply. I found that the ECTD log had a warning at 2023-04-02T00:57:21.859Z, but ETCD did not stop recording the log until 2023-04-02T01:03:10.727Z.

{"level":"warn","ts":"2023-04-02T00:57:21.859Z","caller":"etcdserver/util.go:166","msg":"apply request took too long","took":"712.402169ms","expected-duration":"100ms","prefix":"read-only range ","request":"key:\"by-dev/meta/config\" range_end:\"by-dev/meta/confih\" ","response":"","error":"context canceled"}

Was it because MILVUS responded too slowly that ETCD was closed?

/assign @yanliang567
/unassign
